src/main/java/appeng/me/storage/ExternalStorageFacade.java

```diff
@@ -129,10 +129,17 @@ public int insertExternal(AEKey what, int amount, Actionable mode) {
             int slotCount = handler.getSlots();
             boolean simulate = mode == Actionable.SIMULATE;
 
-            // This uses a brute force approach and tries to jam it in every slot the inventory exposes.
-            for (int i = 0; i < slotCount && !remaining.isEmpty(); i++) {
-                remaining = handler.insertItem(i, remaining, simulate);
-            }
+            int emptySlotsOnlyTries = 1;
+            do {
```

Comment:
This is a terrible idea, it will cause items to be fragmented in anything connected to a storage bus.